com.unity.robotics.urdf-importer/Runtime/Extensions/UrdfGeometryCollision.cs

```diff
@@ -196,16 +204,25 @@ private static void ConvertMeshToColliders(GameObject gameObject, string locatio
                     List<Mesh> colliderMeshes = decomposer.GenerateConvexMeshes(meshFilter.sharedMesh);
                     foreach (Mesh collider in colliderMeshes)
                     {
+                        var c = collider;
                         if (!RuntimeUrdf.IsRuntimeMode())
                         {
                             meshIndex++;
                             string name = $"{filePath}/{templateFileName}_{meshIndex}.asset";
-                            Debug.Log($"Creating new mesh file: {name}");
-                            RuntimeUrdf.AssetDatabase_CreateAsset(collider, name);
-                            RuntimeUrdf.AssetDatabase_SaveAssets();
+                            // Only create new asset if one doesn't exist
+                            if (!RuntimeUrdf.AssetExists(name))
```

Comment:
Also check for check for `settings.OverwriteExistingPrefabs`:
New assets should still be created if the "Overwrite existing prefabs" flag is set during URDF import. For example usecase if the user has modified the stl files and would like to recreate the assets and prefabs.